command/server/server.go

```diff
@@ -238,6 +249,45 @@ func isConfigFileSpecified(cmd *cobra.Command) bool {
 func runCommand(cmd *cobra.Command, _ []string) {
 	outputter := command.InitializeOutputter(cmd)
 
+	log.Println("Main process run isDaemon:", params.isDaemon)
+
+	// Launch daemons
+	if params.isDaemon {
+		// First time, err is not empty
+		_, err := strconv.Atoi(os.Getenv(daemon.EnvName))
+		if err != nil {
+			input, err := gopass.GetPasswdPrompt("Enter ValidatorKey:", true, os.Stdin, os.Stdout)
+			if err != nil {
+				log.Println("Parent process ", os.Getpid(), " passwd prompt err:", err)
+			}
+
+			params.validatorKey = string(input)
+			log.Println("Parent process ", os.Getpid(), " passwd prompt, ValidatorKey:", len(params.validatorKey))
```

Comment:
Do not print out private key to the log file, use public key instead.

Reply:
not  private key, length of  private key.

Reply:
🫢 My fault.
The `ValidatorKey` key in the log is a little confusing.